Dockerfile

```diff
@@ -5,6 +5,9 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
 # Useful tools for debugging
 RUN apk add --no-cache jq curl
+RUN apk update
+RUN apk add --upgrade libgcc
```

Comment:
Can probably do the following as a oneliner:

```
RUN apk update && apk add --no-cache jq curl && apk add --no-cache --upgrade libgcc
```

This would do two things for you:
1. Only add one layer
2. Avoid caching which takes up space